integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/steps/IngressSteps.java

```diff
@@ -60,22 +70,8 @@ public void sendCloudEventToIngressOfBridge(String testBridgeName, String cloudE
                 new Header("ce-subject", "mySubject"));
```

Comment:
Nitpick - the headers are required just for plain request, when cloud event request is provided then the values are taken from request body and no header is required. Though that this would mean a need to specify headers as part of test step.

Reply:
I can differentiate `json` and `plain` cloud event

Reply:
added the `and default headers` as a possible steps for `send cloud event`